index.js

```diff
@@ -0,0 +1,60 @@
+//create footer element
+const currentDate = new Date();
+const thisYear = currentDate.getFullYear();
+const footer = document.createElement("footer");
+const copyright = document.createElement("p");
+copyright.innerHTML =  '<small> mourad fam &copy; 2024</small>';
+
+footer.appendChild(copyright);
+document.body.appendChild(footer);
+
+//create list section 
+
+let skills = ["JavaScript", "HTML", "CSS", "Adobe ", "GitHub"];
+let skillsSection = document.getElementById("skills");
+let skillsList = document.createElement("ul");
+skillsSection.appendChild(skillsList);
+for (let skill of skills){
+  let skillItem = document.createElement("li");
+  skillItem.innerText = skill;
+  skillsList.appendChild(skillItem);
+}
+
+
+
+
+const messageForm =document.querySelector("['name='Leave a Message']");
```

Comment:
This selector is not working.  check the name defined in the html, and make sure messageForm returns value (I add console.log to check)

Reply:
name defined on form is "leave-message"